docs/list.js

```diff
@@ -764,6 +765,7 @@ var list = {
 
 			"场景": {
 				"Fog": "api/zh/scenes/Fog",
+				"FogExp": "api/en/scenes/FogExp", //ZH doc missing for now
```

Comment:
I am not sure temporarily linking to English docs is an acceptable solution.

Reply:
Well,maybe you can just copy the En doc to the corresponding folder in Zh doc, then when the translators find it untranslated, they will directly translate on this file.

Reply:
OK. I will trust that approach, then.